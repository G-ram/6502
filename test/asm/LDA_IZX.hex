a9 40 8d 04 00 a9 04 8d 06 00 a2 02 a9 00 a1 04
09 01 8d 00 01 
