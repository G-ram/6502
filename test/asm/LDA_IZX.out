A: 41
X: 02
Y: 00
P: 00010000
S: 00
PC: 615
100 : 41
101 : 00
